CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Changelog for package foxglove_bridge
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.7.3 (2023-10-23)
+------------------
+* Fix callback accessing invalid reference to promise (`#268 <https://github.com/foxglove/ros-foxglove-bridge/issues/268>`_)
```

Comment:
maybe mention this affects ROS 1 only?

Reply:
Yep, will do. Will have to update this PR anyway because I want to get a regex fix in (will submit that PR shortly)

Reply:
updated to include #270